package.json

```diff
@@ -12,40 +12,44 @@
     "prepublish:next": "npm run build",
     "publish:next": "npm publish --access public --tag next",
     "lint": "eslint \"{src,test}/**/*.ts\" --fix && npm run prettier",
-    "prettier": "prettier --write 'src/*.{ts, js}'"
+    "prettier": "prettier --write \"src/**/*.ts\"",
+    "deps-update": "ncu --packageFile package.json -i"
```

Comment:
Maybe we should use npx to use packages which are expected to be globally installed.
`npx npm-check-updates  --packageFile package.json -i`

Reply:
Updated, please check